docs/backup_workstations.rst

```diff
@@ -91,31 +89,29 @@ mounted and ready to access.
 
 |Backup and TailsData Mounted|
 
-Open a Nautilus window with admin privileges by going to
-**Applications** ▸ **System Tools** ▸ **Terminal**.
+Open a Nautilus window with administrative privileges by first going to
+**Applications** ▸ **System Tools** ▸ **Root Terminal**.
 
-|Open Terminal|
+|Root Terminal|
 
-Type ``gksu nautilus`` at the terminal prompt and hit enter. You'll need to type
-your admin passphrase.
+Type ``nautilus`` at the terminal prompt and hit enter. You'll need to type in
+your administrative passphrase you set at the Tails login screen.
 
-|Start gksu nautilus|
+|Start Nautilus|
 
-.. note::
-  When you run ``gksu nautilus``, you may run into an error where Nautilus
-  complains that it can't create a required folder. If that happens, just click
-  OK and continue normally.
+.. note:: When you run ``nautilus``, you may run into an error where Nautilus
+ complains that it can't create a required folder. If that happens, just
+ click OK and continue normally.
 
-  If a Nautilus window *doesn't* come up, it might be because an admin
-  passphrase wasn't set. If that's the case, you'll need to restart and set an
-  admin passphrase before continuing.
+ If you can't find the "Root Terminal" window, it might be because an
+ administrator passphrase wasn't set when you logged in to Tails. If
+ that's the case, you'll need to restart Tails and set one at the login screen.
 
-.. warning::
-            Make sure you use keep the `Terminal` window open while you perform
-            the backups. Otherwise, the `Nautilus` window will close.
+.. warning:: Make sure you use keep the `Root Terminal` window open while
+ you perform the backups. Otherwise, the `Nautilus` window will close.
 
-Make sure you create a directory on the backup drive to store the data from the
-drive you are backing up:
+ Make sure you create a directory on the backup drive to store the data from the
+ drive you are backing up:
```

Comment:
Remove spaces at the beginning of the lines in this paragraph, it causes it to be interpreted as part of the warning box above